message_sender/factory.py

```diff
@@ -462,4 +479,5 @@ def create_whatsapp_client(cls, channel):
             channel.configuration["API_TOKEN"],
             channel.configuration.get("HSM_NAMESPACE"),
             channel.configuration.get("HSM_ELEMENT_NAME"),
+            channel.configuration.get("MESSAGE_TTL"),
```

Comment:
We need to modify the `__init__` of the `WhatsAppApiSender` to accept this additional argument

Reply:
ohh so add a ttl amongst the other parameters `def __init__(self, api_url, token, hsm_namespace, hsm_element_name, session=None)` ?

Reply:
Yup, since you've added it here after `HSM_ELEMENT_NAME`, it should be added after `hsm_element_name` in `__init__`